src/components/cards/Canvas.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  generateEdge,
+  generateInitialPositions,
+  generateNode,
+} from "@/utils/helpers";
+import { useCallback, useEffect, useMemo, useState } from "react";
+import ReactFlow, {
+  Background,
+  Connection,
+  EdgeChange,
+  Edge,
+  NodeChange,
+  addEdge,
+  applyEdgeChanges,
+  applyNodeChanges,
+} from "reactflow";
+import "reactflow/dist/style.css";
+import { NodeCard } from "./Node";
+
+const CardCanvas = ({ Data, blockHeight }: any) => {
+  const initialEdges = Data.flatMap((element: any, index: any) => {
```

Comment:
Can you change Data into  lowercase and make it more descriptive

Reply:
fixed